bin/ungit

```diff
@@ -31,7 +31,9 @@ const openUngitBrowser = (pathToNavigateTo) => {
 
 const navigate = () => {
   let url = config.urlBase + ':' + config.port;
-  if (config.forcedLaunchPath === undefined) {
+  if (config.forcedLaunchPath === null) {
+    url += '/#/';
+  } else if (config.forcedLaunchPath === undefined) {
     url += '/#/repository?path=' + encodeURIComponent(process.cwd());
   } else if (config.forcedLaunchPath !== undefined && config.forcedLaunchPath !== '') {
```

Comment:
It is easier to just fix this line 😉 . The `undefined` check does not make much sense here since it is already checked above.
Please also make this change in this file
https://github.com/FredrikNoren/ungit/blob/5c01beb7a229f816eb0abcbdabb9cab8e7ee2e6a/public/main.js#L22-L26
which is used by eletron.

```suggestion
  } else if (config.forcedLaunchPath !== null && config.forcedLaunchPath !== '') {
```

---

The `null` check was accidentally removed in https://github.com/FredrikNoren/ungit/pull/1007 and I failed to fix it here https://github.com/FredrikNoren/ungit/pull/1260 because I just tested the command arguments and not the `.ungitrc`.